ovn_cluster.sh

```diff
@@ -438,26 +500,67 @@ function start-db-cluster() {
     $SSL_ARGS start_northd
 
-    ${RUNC_CMD} exec ${CENTRAL_NAME}-2 ${OVNCTL_PATH}  \
+    ${RUNC_CMD} exec ${name}-2 ${OVNCTL_PATH}  \
     --ovn-northd-nb-db=${REMOTE_PROT}:${central_1_ip}:6641,${REMOTE_PROT}:${central_2_ip}:6641,${REMOTE_PROT}:${central_3_ip}:6641 \
     --ovn-northd-sb-db=${REMOTE_PROT}:${central_1_ip}:6642,${REMOTE_PROT}:${central_2_ip}:6642,${REMOTE_PROT}:${central_3_ip}:6642 --ovn-manage-ovsdb=no \
     $SSL_ARGS start_northd
 
-    ${RUNC_CMD} exec ${CENTRAL_NAME}-3 ${OVNCTL_PATH}  \
+    ${RUNC_CMD} exec ${name}-3 ${OVNCTL_PATH}  \
     --ovn-northd-nb-db=${REMOTE_PROT}:${central_1_ip}:6641,${REMOTE_PROT}:${central_2_ip}:6641,${REMOTE_PROT}:${central_3_ip}:6641 \
     --ovn-northd-sb-db=${REMOTE_PROT}:${central_1_ip}:6642,${REMOTE_PROT}:${central_2_ip}:6642,${REMOTE_PROT}:${central_3_ip}:6642 --ovn-manage-ovsdb=no \
     $SSL_ARGS start_northd
 }
 
+function start-ovn-ic() {
+    if [ "$OVN_DB_CLUSTER" = "yes" ]; then
+        central_ic=$(awk -v NAME="${CENTRAL_NAMES[0]}" 'match($0, NAME) {print $2}' _ovn_central_1)
+        central_ic_id="${CENTRAL_NAMES[0]}"-1
+        ${RUNC_CMD} exec ${central_ic_id} ${OVNCTL_PATH}  \
+            --db-ic-nb-create-insecure-remote=yes   \
+            --db-ic-sb-create-insecure-remote=yes start_ic_ovsdb
+    else
+        [ $RELAY_COUNT -gt 0 ] && REMOTE=_ovn_remote_main_db || REMOTE=_ovn_remote
+        central_ic=$(awk -v NAME="${CENTRAL_NAMES[0]}" 'match($0, NAME) {print $2}' $REMOTE |awk -F: '{print $2}')
+        central_ic_id="${CENTRAL_NAMES[0]}"
+        ${RUNC_CMD} exec ${central_ic_id} ${OVNCTL_PATH}  \
+            --db-ic-nb-create-insecure-remote=yes   \
+            --db-ic-sb-create-insecure-remote=yes start_ic_ovsdb
+    fi
+
+    for name in "${CENTRAL_NAMES[@]}"; do
+        if [ "$OVN_DB_CLUSTER" = "yes" ]; then
+            ${RUNC_CMD} exec ${name}-1 ${OVNCTL_PATH}                   \
+                --ovn-ic-nb-db=tcp:${central_ic}:6645                   \
+                --ovn-ic-sb-db=tcp:${central_ic}:6646 start_ic
+            ${RUNC_CMD} exec ${name}-2 ${OVNCTL_PATH}                   \
+                --ovn-ic-nb-db=tcp:${central_ic}:6645                   \
+                --ovn-ic-sb-db=tcp:${central_ic}:6646 start_ic
+            ${RUNC_CMD} exec ${name}-3 ${OVNCTL_PATH}                   \
+                --ovn-ic-nb-db=tcp:${central_ic}:6645                   \
+                --ovn-ic-sb-db=tcp:${central_ic}:6646 start_ic
+        else
+            ${RUNC_CMD} exec ${name} ${OVNCTL_PATH}                     \
+                --ovn-ic-nb-db=tcp:${central_ic}:6645                   \
+                --ovn-ic-sb-db=tcp:${central_ic}:6646 start_ic
+        fi
+    done
+
+    ${RUNC_CMD} exec ${central_ic_id} ovn-ic-nbctl ts-add ts1
+    # wait for ovn-ic to kick in
+    while sleep 2; do
+        ${RUNC_CMD} exec ${central_ic_id} ovn-nbctl ls-list | grep -q ts1 && break
+    done
```

Comment:
This part is not really needed unless we create fake VMs.  It's better to add it there instead.

Reply:
@dceara ack, I will fix it